.gitattributes

```diff
@@ -1,2 +1,3 @@
 # Auto detect text files and perform LF normalization
 * text=auto
+src/requirements/decorated-requirements.json linguist-generated=true
```

Comment:
mark this big file as generated so that github doesn't classify our repo as a json repo lol